zebra-state/tests/basic.rs

```diff
@@ -61,8 +88,11 @@ async fn check_transcripts_testnet() -> Result<(), Report> {
 async fn check_transcripts(network: Network) -> Result<(), Report> {
     zebra_test::init();
 
-    let mainnet_transcript = &[&COMMIT_FINALIZED_BLOCK_MAINNET];
-    let testnet_transcript = &[&COMMIT_FINALIZED_BLOCK_TESTNET];
+    let mainnet_transcript: &[_] = &[
+        &COMMIT_FINALIZED_BLOCK_MAINNET,
+        &GET_TIP_ADD_REVERSED_TRANSCRIPT_MAINNET,
+    ];
+    let testnet_transcript: &[_] = &[&GET_TIP_ADD_REVERSED_TRANSCRIPT_TESTNET];
```

Comment:
Did the revert here but it was pretty weird because we changed a bunch of other things since then. Also, we can never pass the Genesis block strait into the state service via `CommitBlock` because it is by definition finalized, rn it causes errors because it doens't have a parent to trigger its finalization.

Reply:
Can we use CommitFinalizedBlock for genesis?

Should we also have tests that commit blocks in order?
Should we have tests that commit genesis as finalized, then use two `CommitBlock` calls on blocks at height 1 and 2?